.travis.yml

```diff
@@ -1,11 +1,37 @@
-language: python
+language: node_js
 python:
-  - "2.7"
-before_script:
-  - psql -c 'create database oneanddone;' -U postgres
-  - cp oneanddone/settings/local.py-travis oneanddone/settings/local.py
+- "2.7"
+node_js:
+- "0.10"
+addons:
+  artifacts:
+    paths:
+    - results
+    permissions: public-read
+    bucket: fireplace-tests
+before_install:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
 install:
-  - pip install -r requirements.txt
+  - "sudo pip install -r requirements.txt"
+  - "sudo pip install -r oneanddone/functional_tests/requirements.txt"
```

Comment:
Heroku expects the requirements file to be in the root of the project and called `requirements.txt`. I could try using that base file to include other files, but I'm not sure if that's compatible with `peep`, plus I don't really want the test requirements installed on Heroku so I'd also need to do a conditional include, and I'm not sure if that's possible.

So while the bottom line is that I agree it would be nicer to do it as you suggest, I'm not going to allow that to block this effort. It is something I will look into in the future.